templates/filebeat.yml

```diff
@@ -1,7 +1,7 @@
 # WARNING! This file is managed by Juju. Edits will not persist.
 # Edit at your own risk
 filebeat:
-  prospectors:
+  inputs:
```

Comment:
@joedborg, i'm a little nervous that we'll have people that try v6, and if it doesn't work out, want to go back to v5.  Instead of one template to rule them all, what do you think about making both a `filebeat-5.yml` and `filebeat-6.yml`?

We could then choose the right template based on the installed version somewhere around here:

https://github.com/juju-solutions/layer-filebeat/blob/master/reactive/filebeat.py#L59

Filebeat already includes the `apt` layer, so I think it would just require us to [apt.get_package_version('filebeat')](https://github.com/stub42/layer-apt/blob/master/lib/charms/apt.py#L146), parse the major rev, and choose the right template to render.  Whatcha think?

Reply:
yep, that's a great idea @kwmonroe

Reply:
Pushed